assignment 34 completed, topics and posts

droberts84 opened the pull request.

droberts84 commented on app/controllers/summaries_controller.rb:
```diff
@@ -0,0 +1,53 @@
+  def show
+    set_topic
+    set_post
+    set_summary
```
Once you get the summary, is there anyway you could drive the appropriate `post` and `topic` without having to do a database query or calling the `set_topic`, etc.

droberts84 commented:
Oh I didn't mean for you to merge that!  I just made the Pull Request as it
is an easier form for discussion.  I'll help you fix it a bit later today.

On Thu, May 28, 2015 at 8:19 AM Brendan McGivern <notifications@github.com>
wrote:

> Merged #1 <https://github.com/brendanmcgivern/bloccit/pull/1>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/brendanmcgivern/bloccit/pull/1#event-315929798>.
>

brendanmcgivern commented:
Ooops.  I saw that the checkpoint hadn't been checked off yet and then I saw the pull request, so I thought you wanted me to merge and make some alterations based on your previous comments.  Sorry about that.  Date: Thu, 28 May 2015 05:55:30 -0700
From: notifications@github.com
To: bloccit@noreply.github.com
CC: Bmcg0707@hotmail.com
Subject: Re: [bloccit] assignment 34 completed, topics and posts (#1)

Oh I didn't mean for you to merge that!  I just made the Pull Request as it

is an easier form for discussion.  I'll help you fix it a bit later today.



On Thu, May 28, 2015 at 8:19 AM Brendan McGivern <notifications@github.com>

wrote:



> Merged #1 <https://github.com/brendanmcgivern/bloccit/pull/1>.

>

> —

> Reply to this email directly or view it on GitHub

> <https://github.com/brendanmcgivern/bloccit/pull/1#event-315929798>.

>



—
Reply to this email directly or view it on GitHub.

brendanmcgivern commented:
I fixed the the accidental merge.  I reverted the pull request on github and then ran git fetch and git pull in my local environment and everything is back to normal.From: bmcg0707@hotmail.com
To: reply@reply.github.com
Subject: RE: [bloccit] assignment 34 completed, topics and posts (#1)
Date: Thu, 28 May 2015 09:06:17 -0400




Ooops.  I saw that the checkpoint hadn't been checked off yet and then I saw the pull request, so I thought you wanted me to merge and make some alterations based on your previous comments.  Sorry about that.  

Date: Thu, 28 May 2015 05:55:30 -0700
From: notifications@github.com
To: bloccit@noreply.github.com
CC: Bmcg0707@hotmail.com
Subject: Re: [bloccit] assignment 34 completed, topics and posts (#1)

Oh I didn't mean for you to merge that!  I just made the Pull Request as it

is an easier form for discussion.  I'll help you fix it a bit later today.



On Thu, May 28, 2015 at 8:19 AM Brendan McGivern <notifications@github.com>

wrote:



> Merged #1 <https://github.com/brendanmcgivern/bloccit/pull/1>.

>

> —

> Reply to this email directly or view it on GitHub

> <https://github.com/brendanmcgivern/bloccit/pull/1#event-315929798>.

>



—
Reply to this email directly or view it on GitHub.

droberts84 commented:
Cool!  I forgot that GitHub added the revert feature.  Be aware there are
some surprising consequences form using git revert, but they won't effect
you in this situation.  I'll tell you more about it on Friday.

On Thu, May 28, 2015 at 9:58 AM Brendan McGivern <notifications@github.com>
wrote:

> I fixed the the accidental merge. I reverted the pull request on github
> and then ran git fetch and git pull in my local environment and everything
> is back to normal.From: bmcg0707@hotmail.com
> To: reply@reply.github.com
> Subject: RE: [bloccit] assignment 34 completed, topics and posts (#1)
> Date: Thu, 28 May 2015 09:06:17 -0400
>
>
>
>
> Ooops. I saw that the checkpoint hadn't been checked off yet and then I
> saw the pull request, so I thought you wanted me to merge and make some
> alterations based on your previous comments. Sorry about that.
>
> Date: Thu, 28 May 2015 05:55:30 -0700
> From: notifications@github.com
> To: bloccit@noreply.github.com
> CC: Bmcg0707@hotmail.com
> Subject: Re: [bloccit] assignment 34 completed, topics and posts (#1)
>
> Oh I didn't mean for you to merge that! I just made the Pull Request as it
>
> is an easier form for discussion. I'll help you fix it a bit later today.
>
>
>
> On Thu, May 28, 2015 at 8:19 AM Brendan McGivern <notifications@github.com
> >
>
> wrote:
>
>
>
> > Merged #1 <https://github.com/brendanmcgivern/bloccit/pull/1>.
>
> >
>
> > —
>
> > Reply to this email directly or view it on GitHub
>
> > <https://github.com/brendanmcgivern/bloccit/pull/1#event-315929798>.
>
> >
>
>
>
> —
> Reply to this email directly or view it on GitHub.
>
>
>
>
>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/brendanmcgivern/bloccit/pull/1#issuecomment-106322247>
> .
>
